README.md

```diff
@@ -1,6 +1,76 @@
-## README
+# Prior Year Access – File Your State Taxes
 
-### How to compile sass files
-This repo uses the gem `dart-sass` to compile the .scss files. 
+This repository supports **Prior Year Access** for users of **File Your State Taxes**. It allows individuals who filed their state taxes with us in **2023** or **2024** to securely retrieve their tax return PDFs.
 
-To run this locally, use the command: `bin/rake dartsass:build`
+---
+
+## Infrastructure with OpenTofu
+
+This project uses OpenTofu to manage infrastructure as code. It provisions and maintains various cloud resources, including:
+
+- S3 buckets
+- EC2 instances
+- IAM roles
+
+### Usage
+
+Developers interact with OpenTofu using the following commands:
+
+```bash
+tofu init        # Initialize the working directory
+tofu plan        # Preview changes
+tofu apply       # Apply changes to infrastructure
+```
```

Comment:
i don't think these are useful in this repo's readme, but happy to leave it with a note that you must run these from the other repo + within the specific pya folders

Reply:
i agree. i still think the documentation is helpful, but maybe we can move it to a notion doc instead.